tests/PHPStan/Rules/Exceptions/CaughtExceptionExistenceRuleTest.php

```diff
@@ -0,0 +1,29 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Rules\Exceptions;
+
+class CaughtExceptionExistenceRuleTest extends \PHPStan\Rules\AbstractRuleTest
```

Comment:
Extending via a fully qualified name is a part of the coding standard, so this code is also correct.